### PR TITLE
Highlight skipped progress report row on list

### PR DESCRIPTION
**Pull request by tmpacifitech**
Fixes: https://github.com/SeedCompany/cord-field/issues/1023

### CODE REVIEW

File: src/components/PeriodicReports/PeriodicReportsTable.tsx [hunk 1]
```diff
@@ -211,6 +230,16 @@ export const PeriodicReportsTableInContext = ({
...
           thirdSortClick: false,
           draggable: false,
+          rowStyle: ({
+            report: {
+              skippedReason: { value },
+            },
+          }: ReportRow) =>
+            value
+              ? {
+                  color: theme.palette.grey[500],
```
Comment (CarsonF) [inline src/components/PeriodicReports/PeriodicReportsTable.tsx:240]:
```suggestion
                  color: palette.text.disabled,
```

File: src/components/PeriodicReports/ReportLabel.tsx [hunk 2]
```diff
@@ -7,11 +9,21 @@ import {
   SecuredPeriodicReportFragment,
 } from './PeriodicReport.generated';
 
+const useStyles = makeStyles(() => ({
```
Comment (CarsonF) [inline src/components/PeriodicReports/ReportLabel.tsx:12]: We use this component in table rows, in cards, and in titles. What do you think? Does this extra styling make sense in all those places?

File: src/components/PeriodicReports/ReportLabel.tsx [hunk 3]
```diff
@@ -25,7 +37,16 @@ export const ReportLabel = ({
   if (!rep) {
     return null;
   }
-  return <>{getLabel(rep.start, rep.end)}</>;
+
+  const isSkippedProgressReport =
+    rep.__typename === 'ProgressReport' && rep.skippedReason.value;
```
Comment (CarsonF) [inline src/components/PeriodicReports/ReportLabel.tsx:42]: All periodic reports have a skipped prop, not just progress.
```suggestion
    rep.skippedReason.value;
```

File: src/components/PeriodicReports/ReportLabel.tsx [hunk 4]
```diff
@@ -25,7 +37,16 @@ export const ReportLabel = ({
...
+
+  return (
+    <div className={classes.label}>
```
Comment (CarsonF) [inline src/components/PeriodicReports/ReportLabel.tsx:45]: I think span might be more appropriate.
```suggestion
    <span className={classes.label}>
```

File: src/components/PeriodicReports/ReportLabel.tsx (hunk 2 above)
Comment (tmpacifitech) [reply src/components/PeriodicReports/ReportLabel.tsx]: reverted and implemented rendering in `PeriodicReportsTable`.

**Review by CarsonF — APPROVED** [review]
Thanks @tmpacifitech!

**Pull request merged**